workflow/setup_expt.py

```diff
@@ -87,6 +87,15 @@ def _update_defaults(dict_in: dict) -> dict:
     stage_output = f'{inputs.expdir}/{inputs.pslot}/config.stage_ic'
     edit_config(stage_input, stage_output, host_dict, stage_dict)
 
+    # Update fv3jedi-based recentering
+    if yaml_dict.base.DO_JEDIATMENS == "YES":
+        ecen_dict = {
+            "@NMEM_ENS@": inputs.nens
+        }
+        ecen_input = f'{inputs.configdir}/config.ecen_fv3jedi'
+        ecen_output = f'{inputs.expdir}/{inputs.pslot}/config.ecen_fv3jedi'
+        edit_config(ecen_input, ecen_output, host_dict, ecen_dict)
```

Comment:
We shouldn't be handling every file individually.  I know above we did so for `stage_ic`, but will remove that in a upcoming PR.